Portal/View/Components/TextField.swift

```diff
@@ -57,6 +57,14 @@ public struct TextFieldEvents<MessageType> {
         )
     }
     
+    public func toArray() -> [(UIControlEvents, MessageType?)] {
```

Comment:
Also this method should not be here, this is `UIKit` specific and should be added as an extension inside `TextFieldRederer.swift` file.